src/module/utilities.js

```diff
@@ -0,0 +1,36 @@
+const success = () => {
+  const success = document.createElement('div');
+  success.innerText = 'To Do created successfully!';
+  success.classList += 'bg-success success w-50 text-center';
+  const body = document.querySelector('body');
+  body.append(success);
+  setTimeout(() => success.remove(), 3000);
+};
+
+const error = () => {
+  const error = document.createElement('div');
+  error.innerText = 'To Do should be 3-30 characters!';
+  error.classList += 'bg-danger error w-50 text-center';
+  const body = document.querySelector('body');
+  body.append(error);
+  setTimeout(() => error.remove(), 3000);
+};
+
+const createLiTodo = (todoDescription) => {
+  const li = document.createElement('li');
+  const checkbox = document.createElement('input');
+  const description = document.createElement('h6');
+  const dragableI = document.createElement('i');
+  checkbox.type = 'checkbox';
+  description.innerText = todoDescription;
+  // Add classes
+  li.classList += 'list-unstyled d-flex bottom-border w-50 position-relative';
+  checkbox.classList += 'my-3';
+  description.classList += 'mx-3 my-auto';
+  dragableI.classList = 'drag-bg-img my-auto';
+  // Append check box and descripton on li
+  li.append(checkbox, description, dragableI);
+  return li;
+};
```

Comment:
- [ ] When you create each element, consider add an id for each of these. When the elements are listed, they should be scored according to the id for each element. This will help for future steps on this same project when we delete items on it. 🌵
- [ ] After you add the ids for each element, use for, forEach, etc. to list all elements in the project. 🦊

Reply:
All id-s are added now as data-id attributes and I have used forEach to display all elements in the dom.js file where the createLiTodo function is called once for each object in the localStorage by using the returned 'Li' in line 33.